src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -182,7 +183,11 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
         const parsed: number = Math.floor(Number.parseFloat(value) * 60)
         if (
           !Number.isInteger(parsed) || // Check deadline is a number
-          parsed < MINIMUM_ORDER_VALID_TO_TIME_SECONDS || // Check deadline is not too small
+          parsed <
+            (isEthFlow
+              ? // 10 minute low threshold for eth flow
+                DEADLINE_LOWER_THRESHOLD_SECONDS
```

Comment:
If we export this constant and use it out of ETH_FLOW context, it should have ETH_FLOW in the name since its a it confusing here.

Also, it feels wrong to have https://github.com/cowprotocol/cowswap/blob/41b808d940cff8fe492faeb5a2595d7fa473a9ed/src/custom/constants/index.ts#L100 defined in one place (global constants), and the ETH Flow one in some other one (https://github.com/cowprotocol/cowswap/blob/41b808d940cff8fe492faeb5a2595d7fa473a9ed/src/cow-react/modules/swap/state/EthFlow/updaters/deadlineUpdater.tsx#L7). 

It should be next to each other. In fact, since we are doing this, we should do probably another one for the Gnosis Safe. They also need more time. Although in this case, maybe this is more a "default" selection what we want to change, than the "minumum deadline"

Reply:
Moved the EthFlow constant next to the other one